bundles/org.openhab.core.automation.module.script.rulesupport/src/main/java/org/openhab/core/automation/module/script/rulesupport/shared/simple/SimpleTriggerHandlerCallback.java

```diff
@@ -14,12 +14,14 @@
 
 import java.util.Map;
 
+import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.openhab.core.automation.handler.TriggerHandlerCallback;
 
 /**
  *
  * @author Simon Merschjohann - Initial contribution
  */
+@NonNullByDefault
 public interface SimpleTriggerHandlerCallback extends TriggerHandlerCallback {
     public void triggered(Map<String, ?> context);
```

Comment:
I beliebe this is API breaking, right? Before a null context was allowed (even if it was not used in our own classes).

Reply:
Yes it is API breaking. IIRC when `null` is used the `CompositeTriggerHandler` starts throwing NPEs. We can also change that class, but when null is allowed it usually adds more null checks everywhere.

Reply:
I agree with that. You should just add the label :-)

Reply:
Label added!